certbot/certbot/util.py

```diff
@@ -34,7 +34,7 @@
     import distro
 
 if TYPE_CHECKING:
-    import distutils.version
+    import distutils.version # pylint: disable=deprecated-module
```

Comment:
Why is it saying this is deprecated?

Reply:
It's [deprecated since Python 3.10](https://docs.python.org/3/whatsnew/3.10.html#distutils).

Do you mean, why is it complaining now and not in the previous pylint? I'm not sure.

Reply:
The only place we are using this module is our publicly exported function `certbot.util.get_strict_version`, which is slated for removal in Certbot 2.0 and implemented by #9314. I'll leave a comment there to make sure the import is removed.

Reply:
That's great to hear, sounds like a good plan to me.